src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1

```diff
@@ -123,6 +123,64 @@ function Test-AnalysisServicesServer
 	}
 }
 
+<#
+.SYNOPSIS
+Tests scale up and down of Analysis Services server (B1 -> S2 -> S1).
+#>
+function Test-AnalysisServicesServerScaleUpDown
+{
+    param
+	(
+		$location = "West US"
```

Comment:
You shoudls et the location based on the availability of the RP in the location and not to a hard-code value.  Ths prevents using this test against the test cluster, sovereign clouds, etc.